week1/homework/src/server.js

```diff
@@ -10,6 +10,21 @@ function createServer(port) {
 
   const server = http.createServer((request, response) => {
     // TODO: Write your homework code here
+    response.setHeader('Content-Type', 'application/json');
+    if (request.url === '/' || request.url === '/reset') {
```

Comment:
this looks bit buggy
from what i understand
`'/'` should return the current state and not reset it

Reply:
Thank you for feedback . i fixed it .